R/utils.R

```diff
@@ -87,8 +87,8 @@ oc_get_limited <-
   ratelimitr::limit_rate(
     oc_get,
     ratelimitr::rate(
-      n = getOption("oc_requests_per_second", default = 1),
-      period = 1
+      n = 1L,
```

Comment:
Maybe add a comment here as to how that rate can be UPDATE_RATE'd via `oc_config()`, so that future developers might see that more easily?